test/jasmine/tests/config_test.js

```diff
@@ -7,6 +7,7 @@ var click = require('../assets/click');
 var mouseEvent = require('../assets/mouse_event');
 var failTest = require('../assets/fail_test');
 var delay = require('../assets/delay');
+var RESIZE_DELAY = 300;
```

Comment:
cc @antoinerg - bad day for circleci it seems, we were getting repeated test failures here so I encouraged @archmoj to try increasing this delay. At some point we may want to change this so we're waiting on some event (listen for `plotly_afterplot` perhaps?) rather than a fixed delay, that should be more robust (and usually faster!)

Reply:
@antoinerg What value you suggest we use here for `RESIZE_DELAY` now that the CI is back to normal?